src/Microsoft.ML.CodeGen/Templates/Console/ModelProject.tt

```diff
@@ -6,7 +6,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Any reason this moved from core to standard? Does this allow the generated project to exist in a wider set of projects?

Reply:
Because ML.Net target netstandard 2.0 and we want to keep in pace with that. 
@JakeRadMSFT is that correct?

Reply:
Yeah this makes it so .NET Framework and .NET Core projects can reference the model project.